spec/models/ability_spec.rb

```diff
@@ -190,12 +191,18 @@
       it{ should_not be_able_to(:manage, other_event.commercials.first) }
       it{ should be_able_to(:index, my_event.comment_threads.first) }
       it{ should_not be_able_to(:index, other_event.comment_threads.first) }
+
+      it{ should be_able_to(:toggle_user, cfp_role) }
+      it{ should be_able_to(:update, cfp_role) }
```

Comment:
If we check for cfp role, perhaps it makes sense to check for the other roles too? In a shared example?